docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -38,6 +38,50 @@ description: >
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder CLI pertaining to that release.
 {{% /alert %}}
 
+## [v0.22.0](https://github.com/aws/eks-anywhere/releases/tag/v0.22.0)
+
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.32.0 |    ✔    |     -      |    —    |     —      |  —   |
+|      RHEL 8.x (*)   |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     ✔      |    ✔    |     ✔      |  —   |
+
+\* RHEL 8's kernel version 4.18.0 is not supported by Kubeadm 1.32, RHEL 8 support will not be available on Kubernetes 1.32 and above.
+
+### Added
+- Support for Kubernetes v1.32
+- Extended support for Kubernetes versions ([#6793](https://github.com/aws/eks-anywhere/issues/6793), [#4174](https://github.com/aws/eks-anywhere-build-tooling/pull/4174), [#9112](https://github.com/aws/eks-anywhere/pull/9112), [#9115](https://github.com/aws/eks-anywhere/pull/9115), [#9150](https://github.com/aws/eks-anywhere/pull/9150), [#9209](https://github.com/aws/eks-anywhere/pull/9209), [#9218](https://github.com/aws/eks-anywhere/pull/9218), [#9222](https://github.com/aws/eks-anywhere/pull/9222), [#9225](https://github.com/aws/eks-anywhere/pull/9225))
+- Support for deploying EKS-A clusters across vSphere Failure Domains. Available behind feature flag `VSPHERE_FAILURE_DOMAIN_ENABLED` ([#9239](https://github.com/aws/eks-anywhere/pull/9239))
+- Enable hardware Provisioning through ISO booting for baremetal Provider ([#9213](https://github.com/aws/eks-anywhere/pull/9213))
```

Comment:
Can we say,

Enable hardware provisioning through ISO booting for bare metal provider ([#9213](https://github.com/aws/eks-anywhere/pull/9213)). Provides an alternative for customers who do not have L2 connectivity between management and workload clusters as this feature removes the dependency on DHCP for bare metal deployments.